### ISSUE TITLE
TFLearn with Python: An introduction

### ISSUE THREAD

**Issue by MeshiJohn** [section-engineering-education/engineering-education#5820]
### Topic Suggestion
TFLearn with Python: An introduction
#### Proposed article introduction
TFlearn is a modular and transparent Tensorflow deep learning framework. It was created to provide a higher-level API to TensorFlow, while staying transparent and compatible.

You should know TensorFlow. Google's TensorFlow is an open-source artificial intelligence library based on data flow graphs. This allows us to build large-scale neural networks with n layers. In addition to text-based applications, TensorFlow is the greatest library for machine learning and deep learning applications. DeepFace, Facebook's image recognition system, and Siri, Apple's voice recognition technology, both use TensorFlow. TensorFlow is a low-level library. Directly using TensorFlow is tricky due to the API's verbosity and subtleties. TensorFlow has many bugs. TFLearn, a wrapper framework, was created to overcome these issues.

In this essay, I will introduce TFLearn.
#### Key takeaways 
1. Loading the data
2. Standardize Data
3. Labels Hot-Encoding
4. Build the model using TFLearn
#### Article quality
We will make a TFLearn model and use it to predict the labels in the test data set that I will give us. Codes are shown in the article to make it easier to understand, and to keep people on the page longer. The article is meant to be concise and clear to the point.
#### References
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by WanjaMIKE** [comment]
Introductory aspects to TensorFlow seem to be adequately covered in this articles https://github.com/section-engineering-education/engineering-education/issues?q=tensorflow+ Feel free to search for another topic. Thanks

**Comment by MeshiJohn** [comment]
> Introductory aspects to TensorFlow seem to be adequately covered in this articles https://github.com/section-engineering-education/engineering-education/issues?q=tensorflow+ Feel free to submit another topic. Thanks

@WanjaMIKE The link you provided takes me back to this topic.
